release-1.8/release-notes-draft.md

```diff
@@ -213,6 +213,18 @@ This section contains a list of known issues reported in Kubernetes 1.8 release.
 
 For more information, see [#1996](https://github.com/kubernetes/minikube/issues/1996).
 
+* Audit logs might impact the API server performance and the latency of large request and response calls. The issue is observed under the following conditions: feature gate for advanced audit logging is enabled, audit logging uses the log backend in JSON format, and the audit policy records large events at request and response level.
+
+For more information, see [#51899](https://github.com/kubernetes/kubernetes/issues/51899).
+
+* The `ENABLE_APISERVER_BASIC_AUDIT` flag is depreciated. Therefore, the basic audit does not work.
```

Comment:
Remove `therefore`, as this will likely be fixed in 1.8.1. Maybe:
```
The `ENABLE_APISERVER_BASIC_AUDIT` configuration parameter for GCE deployments is broken, but deprecated.
```

/cc @crassirostris